MiscVBAFunctions/Modules/MiscTables.bas

```diff
@@ -191,3 +191,142 @@ Function TableColumnToArray(TableDicts As Collection, ColumnName As String) As V
     TableColumnToArray = arr
 End Function
 
+
+Function TableColumnToCollection(TableDicts As Collection, ColumnName As String) As Collection
+    ' Append the selected key's value from each Dict in the input Collection to a Collection
+    '
+    ' Args:
+    '   TableDicts: A collection of Dicts.
+    '   ColumnName: Name of the column that will be returned as a Collection.
+    '
+    ' Returns:
+    '   Collection of the selected column.
+    
+    Dim col1 As Collection
+    Dim dict As Dictionary
+    
+    Set col1 = New Collection
+    For Each dict In TableDicts
+        col1.Add dictget(dict, ColumnName)
+    Next dict
+    
+    Set TableColumnToCollection = col1
+End Function
+
+
+Public Sub ResizeLO(SelectedLO As ListObject, NumRows As Long)
+    ' Resize a table to the desired number of data rows. The columns remain unchanged.
+    ' If NumRows is set to "0", the table will instead be resized to "1" and the
+    ' content in that 1st row will be cleared.
+    '
+    ' Args:
+    '   SelectedLO: Selected List Object.
+    '   NumRows: Number of desired rows.
+    
+    Dim oldNumRows As Long
+    oldNumRows = SelectedLO.ListRows.Count
+    
+    ' Don't allow tables to be resized to zero rows.  Excel won't really do
+    ' this anyway - it's possible to get the DataBodyRange to be empty, in
+    ' which case the single row displayed in the table is the Insert row, but
+    ' this creates more problems than it solves.  Instead, resize the table to
```

Comment:
I prefer to set the `DataBodyRange` to nothing when there are no rows in a ListObject, because that's what it means, 0 rows. The problem with keeping one "row" with no data is if we do `ListObject.ListRows.Add` then there will be two rows: (1) is the "empty" row and (2) is the new one that was added.

Reply:
> this creates more problems than it solves.

What is meant by this? If it causes problems elsewhere (ie other MiscVBA functions) it should be dealt with there, because those functions will receive list objects with no `DataBodyRange`. See comments in `GetColumnLO` below: https://github.com/VirtualActuary/MiscVBAFunctions/pull/66/files#r925995903